File: WEEK 2/week 2 hello bs ece.js
using System;
using System.Threading;

class Program {
  static void Main() {
    Console.WriteLine("BS ECE 1-2 DEMO");
    for (int i = 10; i > -1; i--) {
      Console.WriteLine(i);
      Thread.Sleep(1000);

    }
  }
}

File: WEEK 2/week 2 hello bs ece.java
public class myfirstcode {
  public static void main(String[] args) {
    System.out.println("BS ECE 1-2 demo");
    for (int i = 10; i > -1; i--) {
      System.out.println(i);
      try {
        Thread.sleep(1000);
      } catch (InterruptedException ex) {}
    }
  }
}


from time import sleep


   